src/components/Stack/tests/Stack.test.tsx

```diff
@@ -0,0 +1,254 @@
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.prop('distribution')).toBe('trailing');
+        });
+
+        it('should verify distribution property when set as center', () => {
+            const stackWrapper = mount(
+                                        <Stack distribution="center">
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.prop('distribution')).toBe('center');
+        });
+
+         it('should verify distribution property when set as fill', () => {
+            const stackWrapper = mount(
+                                        <Stack distribution="fill">
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.prop('distribution')).toBe('fill');
+        });
+
+         it('should verify distribution property when set as equalSpacing', () => {
+            const stackWrapper = mount(
+                                        <Stack distribution="equalSpacing">
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.prop('distribution')).toBe('equalSpacing');
+        });
+
+        it('should verify distribution property when set as fillEvenly', () => {
+            const stackWrapper = mount(
+                                        <Stack distribution="fillEvenly">
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.prop('distribution')).toBe('fillEvenly');
+        });
+
+         it('should verify distribution property is not defined', () => {
+            const stackWrapper = mount(
+                                        <Stack>
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.prop('distribution')).toBeUndefined();
+        });
+    });
+
+    describe('Test stack with item', () => {
+        it('should verify stack where a single item fills the remaining space', () => {
+            const stackWrapper = mount(
+                                        <Stack >
```

Comment:
you don't need the space between the k and the >. This applies to all the components with the extra space.